lib/src/authentication/widgets/input_password.dart

```diff
@@ -94,7 +108,9 @@ class _InputPasswordState extends State<InputPassword> {
         validator: (String? value) => _validate(value, l10n),
         obscureText: isPasswordObscured,
         decoration: InputDecoration(
-          hintText: l10n.input_password_placeholder,
+          hintText: widget.passwordToConfirmController != null
+              ? l10n.input_password_confirmation_placeholder
+              : l10n.input_password_placeholder,
```

Comment:
If we are to extract those widgets (`InputEmail`, `InputPassword`, `AppSpacer`, etc.) into their library, the `hintText` should be available in the parameters of the widget. By default, the hint text would be `input_password_placeholder` and the confirmation would be passed by the parent. What do you think?

Reply:
It makes sense
Updated